services/TipiVariable.py

```diff
@@ -104,11 +119,11 @@ def processRequest(self, message):
             tmp = []
         
             try:
-                session_id = self.ti_vars['SESSION_ID']
+                session_id = self.ti_global['SESSION_ID']
```

Comment:
Now the app cannot control if SESSION_ID is global or not... If coder does want to set up their own server, they won't be able to use this. And REMOTE_HOST etc must be global..

Reply:
I don't see anywhere that REMOTE_HOST can be set in globals...